agent/helpers.go

```diff
@@ -98,11 +97,12 @@ func (h Helper) isRouteExist(ip net.IP, netmask string) error {
 }
 
 // createRoute creates IP route, returns nil if success and error otherwise.
-func (h Helper) createRoute(ip net.IP, netmask string, via string, dest string) error {
+func (h Helper) createRoute(ip net.IP, netmask string, via string, dest string, extraArgs ...string) error {
```

Comment:
I was referring to the value, rather than the name.
In one case, the caller passes the word "via", and in the other it passes the word "dest".